gemd/entity/bounds/categorical_bounds.py

```diff
@@ -80,7 +79,7 @@ def union(self,
 
         Parameters
         ----------
-        others: Union[CategoricalBounds, CategoricalValue]
+        others: Union[CategoricalBounds, ~gemd.value.categorical_value.CategoricalValue]
```

Comment:
Nitpick?
In some places in the docs, you use `Union` when a parameter can accept multiple types. In others, such as `gemd/builders/impl.py` above, you use "or". Is there an intended distinction?